func_adl_xAOD/common/cpp_representation.py

```diff
@@ -182,6 +182,10 @@ def get_element_type(self):
         'Return the type of the element of the sequence'
         return cast(ctyp.collection, self.cpp_type()).element_type
 
+    def token_type(self):
+        'Return the type of the token required, for CMS miniAOD specifically at this moment'
+        return None
```

Comment:
Mention what the "token" is here (what it is used for) - otherwise this doesn't give the reader any reference.

Reply:
It doesn't say what the token is yet - it just says cms will need it. But you can mention that it is used for lookup in the event data.